lib/net.js

```diff
@@ -541,7 +541,7 @@ ObjectDefineProperty(Socket.prototype, 'readyState', {
 ObjectDefineProperty(Socket.prototype, 'bufferSize', {
   get: function() {
     if (this._handle) {
-      return this[kLastWriteQueueSize] + this.writableLength;
+      return this.writableLength;
```

Comment:
I don’t think we need to guard this on `if (this._handle)` anymore.

Reply:
There are some tests that expect undefined when there is no handle.

Reply:
Changing that would probably make this semver-major?

Reply:
Hm yeah, I guess.